frontend/cypress/integration/featureFiles/waypoint.feature

```diff
@@ -37,7 +37,8 @@ Feature: Kiali Waypoint related features
     And the link for the waypoint "waypoint" should redirect to a valid workload details
 
   Scenario: [Workload details - waypoint] The workload details for a waypoint are valid
-    Given user is at the details page for the "workload" "bookinfo/waypoint" located in the "" cluster
+    Given the waypoint is healthy
```

Comment:
I think it should be applied to all the waypoints. There is a hack script that installs ~= 8 waypoints in 8 different namespaces to validate the waypoints logic (All of them in this file)

Reply:
I moved the check to the feature file preamble (or whatever it's called) so that it runs before each test. I also updated it to check for all the waypoints. It seemed like: `gateway.istio.io/managed=istio.io-mesh-controller` was the only consistent label used across all waypoints.